projects/commudle-admin/src/app/feature-modules/public-community/components/speakers/speakers.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="speakers-container d-flex flex-wrap justify-content-center ">
+  <nb-card *ngFor="let speaker of user">
+    <a class="clickable" [routerLink]="['/users', speaker.username]">
+
+      <nb-card-body>
+
+        <div class="user-card">
+          <div class="banner">
+            <img *ngIf="speaker.profile_banner_image.url != undefined" src="{{ speaker.profile_banner_image.url}}">
```

Comment:
We can skip checking undefined, it can simply be 
` *ngIf = speaker.profile_banner_image `

Also, instead of url (which is the full size image), we will use i350 (speaker.profile_banner_image.i350), it's a smaller size image and loads faster